nss-day-cohort-39/holiday-road-cosmic-chameleons
# added details button modified index parkjs mainjs parkdialogjs

### Testing

1. In master do a git fetch --all
2. cd into mjm-details-button and launch code
3. In separate tab in the terminal cd in api folder and launch json with json-server -p 8090 -w parks.json
4. select an park
5. scroll down and make sure an park appears
6. then click the detail button and make sure the description populates in modal
7. close modal with ESC

### Files Used
parks/park.js
parks/ParkDialog.js
main.js
index.html
